playbooks/common/openshift-cluster/upgrades/files/rpm_versions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+while getopts ":c" opt; do
+  case $opt in
+    c)
+      echo "-c was triggered!" >&2
+      containerized="TRUE"
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      ;;
+  esac
+done
+
+if [ "${containerized}" == "TRUE" ] ; then
+  docker exec atomic-openshift-master rpm -q atomic-openshift 
```

Comment:
it looks like the `rpm -q` should be using `$@` instead, and this will probably need to be refactored to handle a parameter for the image name to use.